explos/channel-manager/sqlLite.go

```diff
@@ -81,6 +81,7 @@ func (s sqlLite) SyncTeritoriKey(teritoriPubkey string, bertyPubkey string) erro
 	return nil
 }
 
+// AddChannel to database if it did not already exist
```

Comment:
what do you think about write every comment in the present tense ?
-  it will be more compliant
- this will answer the format "what does this function ?" -> comment.